rudder-webapp/SPECS/rudder-webapp.spec

```diff
@@ -406,24 +406,15 @@ fi
 
 %if 0%{?rhel} || 0%{?fedora}
 # SELinux support
-# Check "sestatus" presence, and if here, probe if SELinux
-# is enabled. If so, then tweak our installation to be
+# Check "sestatus" presence, and if here tweak our installation to be
 # SELinux compliant
 if type sestatus >/dev/null 2>&1
 then
-  if [ $(LANG=C sestatus | grep -cE "SELinux status:.*enabled") -ne 0 ]
-  then
-
-    # Adjust the inventory directories SELinux context
-    chcon -R --type=httpd_sys_content_t /var/rudder/inventories/incoming
-    chcon -R --type=httpd_sys_content_t /var/rudder/inventories/accepted-nodes-updates
-
-    # If necessary, add the rudder-webapp SELinux policy
-    if [ $(semodule -l | grep -c rudder-webapp) -eq 0 ]
-    then
-      semodule -i /opt/rudder/share/selinux/rudder-webapp.pp
-    fi
-  fi
+  # Add/Update the rudder-webapp SELinux policy
+  semodule -i /opt/rudder/share/selinux/rudder-webapp.pp
+	
+  # Ensure inventory directories context is set
+  restorecon -R /var/rudder/inventories
```

Comment:
Also, will restorecon work even with SELinux in disabled mode (guess so, but just to be sure)